bpf/lib/encap.h

```diff
@@ -162,13 +162,13 @@ __encap_and_redirect_with_nodeid(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
  */
 static __always_inline int
 encap_and_redirect_with_nodeid(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
-			       __u8 key __maybe_unused, __u32 seclabel,
+			       __u8 encrypt_key __maybe_unused, __u32 seclabel,
 			       __u32 dstid, const struct trace_ctx *trace)
 {
 #ifdef ENABLE_IPSEC
-	if (key) {
-		__u8 encrypt_key = get_min_encrypt_key(key);
+	encrypt_key = get_min_encrypt_key(encrypt_key);
```

Comment:
I would delete an empty line (as we aren't declaring a var anymore) and braces around the single-line if below (kernel-like-style)